api/filters/namespace/namespace.go

```diff
@@ -19,9 +20,28 @@ type Filter struct {
 	// FsSlice contains the FieldSpecs to locate the namespace field
 	FsSlice types.FsSlice `json:"fieldSpecs,omitempty" yaml:"fieldSpecs,omitempty"`
 
+	// SkipExisting means only blank namespace fields will be set
+	SkipExisting bool `json:"skipExisting" yaml:"skipExisting"`
+
+	// SetRoleBindingSubjects determines which subject fields in RoleBinding and ClusterRoleBinding
+	// objects will have their namespace fields set. Overrides field specs provided for these types, if any.
+	// - defaultOnly (default): namespace will be set only on subjects named "default".
+	// - all: namespace will be set on all subjects
+	// - none: all subjects will be skipped.
```

Comment:
One complication I just realized is the meaning of "all". Some options:
1. Support the value, and drive it with `create: false` fieldspecs. This means all subject namespaces with any value will be overwritten, allowing the user to effectively tell us which subjects are namespaced by setting dummy values in them. Has a weird interaction where `skipExisting: true, setRoleBindingSubjects: all` sets nothing. 
1. Support the value, and drive it with `create: true` fieldspecs. I think this is unacceptable, because it will add namespaces to subjects with cluster-scoped kinds.
1.  Don't support that value, but stop stripping the role binding fieldspecs so that users can do one of the above themselves (not any more elegantly).
1. Don't support that value, and continue stripping the role binding fieldspecs.
1. Support the value, and drive it with openAPI-aware code. Much the same as we do for metadata.namespace, except only do it if the "all" option is specified *or* the fieldspecs are given.
 
This PR currently does the first one. But I'm leaning towards one of the last two after typing this out.

Reply:
Sorry for the late response. I agree that the last two options are best, and am happy with the choice of option 5.